src/service/user/settingsAdapter.js

```diff
@@ -0,0 +1,20 @@
+import { CHALLENGE_TYPES } from '@/constants';
+
+function getChallengeType(settings) {
```

Comment:
We have the same adapter in `authAdapter`. Move it to the global adapters

Reply:
they are not the same, changed only with checking field names, and returned different structures.
or what is your solution?

Reply:
If they have any differences - ok, if they are the same - just use one function in the both cases